lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -903,6 +908,7 @@ def test_render_html_view_invalid_certificate_configuration(self):
         self.course.cert_html_view_enabled = True
         self.course.save()
         self.store.update_item(self.course, self.user.id)
+        CourseOverview.load_from_module_store(self.course.id)
```

Comment:
I believe the alternative to updating each of these tests is to set the following test-class variable:
```
ENABLED_SIGNALS = ['course_published']
```

Reply:
I was wondering why this signal wasn't firing on my commute last night. Thanks!

Reply:
Yeah, I added signal isolation in #14165 to help speed up tests -- we create courses all over the place and a lot of post-publish machinery triggers off of that, even though it's almost never used. Suppressing signals by default shaved off something like a third of the execution time across all tests.

Still, it's not very discoverable, and our docs for modulestore testing could use a lot of work. Just out of curiosity, when you went to create this test, did you start from documentation, or by examining an existing test case in another app?

Reply:
@ormsbee makes sense. I didn't actually write this particular test. But for recent tests I've written that have created courses I started by examining existing tests.